test-fuzz/verify.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+/*!
+ * verify.js - verify script for fuzz testing
+ * Copyright (c) 2018, the bcoin developers (MIT License).
+ * https://github.com/bcoin-org/bcoin
+ */
+
+'use strict';
+
+const bcash = require('..')
+const fs = require('fs')
+
+const filename = process.argv[2];
+
+fs.readFile(filename, 'utf8', (err, data) => {
+  if (err) {
+    console.error(err)
+    process.exit(2);
+  }
+
+  try {
+    const lines = data.split('\n');
+    const outputScriptBuffer = Buffer.from(lines[0], 'hex');
+    const txBuffer = Buffer.from(lines[1], 'hex');
+
+    const tx = bcash.TX.fromRaw(txBuffer);
```

Comment:
Does it make sense to pull out the modules we need from the library at the top of the file. 
I think this is what we're doing in other places:
```
const TX  = ('../lib/primitives/tx');
const Script = ('../lib/script');
```

Reply:
Updated